docs/manual.tex

```diff
@@ -0,0 +1,59 @@
+\documentclass{article}
+\usepackage{geometry}
+ \geometry{
+ a4paper,
+ total={170mm,257mm},
+ left=20mm,
+ top=20mm,
+ }
+\usepackage{tgheros}
+\usepackage[utf8]{inputenc}
+\usepackage[english]{babel}
+%\usepackage[english]{isodate}
+%\usepackage[parfill]{parskip}
+\usepackage[hybrid]{markdown}
+\markdownSetup{
+  pipeTables,
+  tableCaptions,
+  rendererPrototypes = {
+    image = {\begin{center}\setkeys{Gin}{width=.99\linewidth}\includegraphics{#2}\end{center}}, % center images inline expanding to page width
+    codeSpan = {\texttt{#1}}, % Render inline code via `\texttt`.'
+  }
+}
+\title{AmpliPi Home Audio Controller - User Manual}
```

Comment:
Amplipro, also you might want to play with this title page so it looks a little better.

You should be able to set a different header font and change the font for everything else in here too, I think Raleway medium for the header and maybe Roboto or something else sans serif for everything else would look good.